test/integration/features/cmd/version.feature

```diff
@@ -0,0 +1,7 @@
+@minishift-only @command
+Feature: Version command
+Command "minishift version" shows version of minishift binary.
+
+  Scenario: Setup-cdk installs and enables specific addons
```

Comment:
I guess this is a cut&paste error. Let's have a correct scenario description here.